internal/cri/server/service.go

```diff
@@ -180,6 +185,87 @@ type CRIServiceOptions struct {
 	Client *containerd.Client
 }
 
+func (c *criService) initTracing() error {
+	if c.config.Tracing == nil {
```

Comment:
Since we remove that function about loading from env, please update the log. 
it should be tracing is disabled.